src/main/interface/msp.c

```diff
@@ -657,12 +660,15 @@ bool mspCommonProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFnPtr
     }
 
     case MSP_BATTERY_CONFIG:
-        sbufWriteU8(dst, batteryConfig()->vbatmincellvoltage);
-        sbufWriteU8(dst, batteryConfig()->vbatmaxcellvoltage);
-        sbufWriteU8(dst, batteryConfig()->vbatwarningcellvoltage);
+        sbufWriteU8(dst, (batteryConfig()->vbatmincellvoltage + 5) / 10);
```

Comment:
why changing the values?

Reply:
because these tree are the legacy values (uint8_t), while vbatmincellvoltage is an uint16_t so we have to transform a 370 in 37, for example. This way a legacy receiver will receive the correct value.

Below we write the values with full precision!

Reply:
This is not an idea of mine, I took from the cherry-pick :D

Reply:
ah i get it know thank you

Reply:
don't touch msp please without adding numlber in api version 
it breaks allthe stuff  for the gui  ... you can change everything in the cide execpt the msp 
@tylercorleone

Reply:
> the root cause is the configurator being broken and not getting maintained. there is no actual relation but some people trying to manage "user experience" in a strange way. flight code is flight code and this is not the configurator repository.
> besides, this project should have had a roadmap ages ago. so people can actually agree on something instead of acting in their own preference.

well msp is not flight code ... it is the contract between gui and firmware it have to be done as a road map as you said ;-)

Reply:
@loutwice I have to increment API_VERSION_MINOR in this case? Or it's better to rollback the changes to the MSP?